app/src/ui/changes/commit-message.tsx

```diff
@@ -569,19 +598,29 @@ export class CommitMessage extends React.Component<
 
   private renderPermissionsCommitWarning() {
     const {
+      commitToAmend,
       showBranchProtected,
       showNoWriteAccess,
       repository,
       branch,
     } = this.props
 
-    if (showNoWriteAccess) {
+    if (commitToAmend !== null) {
       return (
-        <PermissionsCommitWarning>
+        <CommitWarning icon={CommitWarningIcon.Information}>
+          Your changes will be applied to your{' '}
+          <strong>most recent commit</strong>.{' '}
+          <LinkButton onClick={this.onStopAmending}>Stop amending</LinkButton>{' '}
+          to go back to normal.
```

Comment:
From @ampinsk's suggestion from workshopping in Slack.
```suggestion
          Your changes will modify your{' '}
          <strong>most recent commit</strong>.{' '}
          <LinkButton onClick={this.onStopAmending}>Stop amending</LinkButton>{' '}
          to make these changes as a new commit.
```